src/seedu/addressbook/data/person/NameTest.java

```diff
@@ -0,0 +1,44 @@
+package seedu.addressbook.data.person;
+
+import static org.junit.Assert.*;
```

Comment:
in general, import only what is necessary - can use Eclipse's organise imports to do this automatically